src/lib/cache/subCache.cpp

```diff
@@ -211,6 +211,20 @@ void subCacheInit(bool multitenant)
 
 /* ****************************************************************************
 *
+* subCacheDisable -
+*
+*/
+#ifdef UNIT_TEST
+void subCacheDisable(void)
```

Comment:
Why not put this function in unittest.cpp (or some other test file) ?
subCacheActive is a global variable, not static ...